package.json

```diff
@@ -30,8 +30,9 @@
     "react-dom": ">= 16.6.1",
     "styled-components": ">= 5.X"
   },
+  "devDependencies_comment": "Hey there, is Chris from the past. If you're in the future and this issue is fixed: https://github.com/storybookjs/storybook/issues/11255 then you can probably remove core-js from the dependencies.",
```

Comment:
Reading through the thread, I see that they've fixed the issue in an alpha release.

- https://github.com/storybookjs/storybook/pull/13055
- https://github.com/storybookjs/storybook/releases/tag/v6.2.0-alpha.0

The following gets me a working/built storybook. It's alpha, but I'm okay with that since it's strictly a dev tool, it fixes the problem now, it works, and it's removes the need try and remember that this note is in the package.json file.

```
npm uninstall core-js
npm i -D storybook@6.2.0-alpha.1
npm run storybook:build
open storybook-static/index.html
```

Also, if you're updating storybook, I was able to update all the addons and get a successful build.

```
npm i -D @storybook/addon-a11y@6.1.9
npm i -D @storybook/addon-actions@6.1.9
npm i -D @storybook/addon-knobs@6.1.9
npm i -D @storybook/addon-links@6.1.9
npm i -D @storybook/addon-storysource@6.1.9
npm i -D @storybook/addon-viewport@6.1.9
npm i -D @storybook/addons@6.1.9
npm i -D @storybook/react@6.1.9
```

Reply:
Yep, that worked for me! Pushed the change.